common/log/platform/boost/boost_log.cpp

```diff
@@ -184,6 +187,31 @@ void SetLevel(LogLevel level) {
 	global_logger_.SetLevel(level);
 }
 
+error::Error SetupFileLogging(const string &log_file_path, bool exclusive) {
+	typedef sinks::synchronous_sink<sinks::text_ostream_backend> text_sink;
+	boost::shared_ptr<text_sink> sink = boost::make_shared<text_sink>();
+
+	// Add a stream to write log to
+	auto log_stream = boost::make_shared<std::ofstream>(log_file_path);
```

Comment:
Should we have a short discussion on whether to use the rotating log-file backend?

I'm a little undecided. I _think_ we do in the old client, but I would have to double check.

Reply:
I wasn't referring to the deployment rotation.

However, I did go and check the regular log-file functionality, and it does not seem we rotate it, just dump everything to file, so this is fine .